lib/index.ts

```diff
@@ -1,22 +1,46 @@
+/* global QUnit */
 import { TestModel } from '@xstate/test';
+import { TestPlan } from '@xstate/test/lib/types';
 
 export function setupXStateTest<TestContext, TContext>(
   hooks: NestedHooks,
   testModel: TestModel<TestContext, TContext>
 ) {
-  hooks.after(async function(assert) {
+  hooks.after(function(assert) {
     try {
-      await testModel.testCoverage();
+      testModel.testCoverage();
     } catch (error) {
       assert.ok(false, 'Not all tests were exercised in this test suite (see console)');
-      console.info([
-        'Tests in this file are generated. It is checking for full coverage of all possible test',
-        'paths. However, if you have filtered or scoped (with module=...) then this assertion will be',
-        'a false negative and is safe to ignore.',
-        '',
-        'For more details: https://github.com/davidkpiano/xstate/tree/master/packages/xstate-test#testmodeltestcoverageoptions'
-      ].join('\n'));
+      console.info(
+        [
+          'Tests in this file are generated. It is checking for full coverage of all possible test',
+          'paths. However, if you have filtered or scoped (with module=...) then this assertion will be',
+          'a false negative and is safe to ignore.',
+          '',
+          'For more details: https://github.com/davidkpiano/xstate/tree/master/packages/xstate-test#testmodeltestcoverageoptions',
+        ].join('\n')
+      );
       console.error(error);
     }
   });
 }
+
+type TestCallbackFn<TestContext, TContext, TReturn> = (
+  assert: Assert,
+  path: TestPlan<TestContext, TContext>['paths'][0]
+) => TReturn;
+
+export const testShortestPaths = <TestContext, TContext, TReturn>(
```

Comment:
I added this because it helped me look at the tests and focus on only what I'm testing.

We can add more helpers for the other testModel.* methods later

Reply:
Would this abstraction hide valuable understanding? I hides the API people would be used to when reading/learning the @xstate/test docs. Then again this addon is already opinionated so there would be an expectation it knows how to do the QUnit things.

Reply:
yeah, that's the risk with any abstraction. Based on the 3 or so uses I've seen with qunit and xstate, I don't think there is much to be done with what the abstraction is hiding. idk